src/SourceBuild/content/eng/tools/tasks/PackageDiff/GetClosestPackage.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Threading.Tasks;
+using Microsoft.Build.Framework;
+
+public class GetClosestPackage: GetClosestPackageBase
+{
+    [Required]
+    public string PackagePath { get; set; } = "";
+
+    [Output]
+    public string ClosestPackagePath { get; set; } = "";
+
+    public override async Task<bool> ExecuteAsync()
+    {
+        ClosestPackagePath = await GetClosestPackage(PackagePath);
+        return true;
+    }
+}
+
+public class GetClosestPackages: GetClosestPackageBase
```

Comment:
Why not just have the task that accepts multiple packages and remove the singular one?